srtcore/channel.cpp

```diff
@@ -219,7 +219,9 @@ void CChannel::setUDPSockOpt()
          }
          else //Assuming AF_INET6
          {
+#ifdef IPV6_TCLASS
             if(0 != ::setsockopt(m_iSocket, IPPROTO_IPV6, IPV6_TCLASS, (const char*)&m_iIpToS, sizeof(m_iIpToS)))
+#endif
                throw CUDTException(MJ_SETUP, MN_NORES, NET_ERROR);
```

Comment:
Here it will throw error if IPV6_TCLASS is not available but m_iIpTTL is not default for an IPv6 connection. Is this intended?

Reply:
Well the function failure throws. So if figured this is a function failure if the operation is not supported. So I figured it was within the spirit of the implementation. I can change it if you want.

Reply:
Makes sense. Just wanted to double-check if this was the intention.